LandSandBoat/server
# [SMN] Correct Carbuncle animations, BP abilities, implement Soothing Ruby and Holy Mist, various fixes

**_I affirm:_**

## What does this pull request do?



Updates all of Carbuncles animations to reflect new pet_skills table
Compared all Blood pact msg's to retail captures
Removes Carbuncles animation shifting from core
Implements Soothing Ruby based on retail captures
Implements Holy Mist as a slightly stronger version of BP Rage "Meteor"
Various fixes found along the way:

- Correction to uint8 that caused issues displaying level 99 BP ability (WinterS)
- Correction to charUtils to allow usage of new BP ability (WinterS)

## Steps to test these changes


Use Carbuncle BP Ward to see proper messages displayed
Use Carbuncle BP Rage to see proper messages displayed
Verified uint32 change did not affect any additional SMN BP of other summons
Verified status effect removal from Soothing Ruby based on skill (retail captures referenced)
Verified Holy Mist to be a slightly increased version of BP Rage "Meteor"
